src/main/c/trufflemock/trufflemock.c

```diff
@@ -0,0 +1,309 @@
+/*
```

Comment:
This file should be generated from `truffle.h`, it should be relatively straightforward with a small Ruby script.
For now, I think the re-generation can be manual. But maybe the `extconf.rb` could just re-generate every time.